tests/unit/stats/test_measure_to_view_map.py

```diff
@@ -229,12 +221,11 @@ def test_record(self):
         view_columns = ["testTag1", "testColumn2"]
         view_measure = measure
         view_aggregation = mock.Mock()
-        view = View(
-            name=view_name,
-            description=view_description,
-            columns=view_columns,
-            measure=view_measure,
-            aggregation=view_aggregation)
+        View(name=view_name,
```

Comment:
Why is this created but never used?

Reply:
I didn't start the fire here, I just removed the unused `view` var to fix the [F841](https://lintlyci.github.io/Flake8Rules/rules/F841.html) lint error, and same for the changes below.

Many of the tests in this module seem half-baked, but I think this is a problem for another PR.

Reply:
SGTM